llvm/include/llvm/IR/DebugLoc.h

```diff
@@ -115,6 +138,23 @@ namespace llvm {
     DebugLocKind getKind() const { return Loc.Kind; }
 #endif
 
+#if ENABLE_DEBUGLOC_ORIGIN_TRACKING
+#if !ENABLE_DEBUGLOC_COVERAGE_TRACKING
+#error Cannot enable DebugLoc origin-tracking without coverage-tracking!
+#endif
```

Comment:
Not sure how critical checking this case is... IIUC, CMake options won't let you set such anything anyway, so perhaps remove this?

If keeping it, perhaps remove the blank line after this `#endif` here.

Reply:
I figured it'd be worth keeping something in the source just in case of some weird scenario with leaky environment variables; it's not likely to happen, but this will give a much cleaner error if it does.

Reply:
Hmm, how are environment variables ending up with up confused compiler defines...? Anyway, it's fine to keep, even if highly unlikely to be hit.